lib/ifttt.js

```diff
@@ -0,0 +1,19 @@
+const {config} = require('./config');
+
+const request = require('request-promise').defaults({
+    method: 'POST',
+    baseUrl: 'https://maker.ifttt.com/trigger'
+});
+
+module.exports = {
+    notifyIFTTT
+};
+
+function notifyIFTTT(message) {
+    return request({
+        url: `/${config.get('notifications.ifttt.webhookEvent')}/with/key/${config.get('notifications.ifttt.webhookKey')}`,
+        json: {
+            value1: message
```

Comment:
Where does the name `value1` come from? Is this expected by IFTT? Do you have a link to the docs?